CLOUDP-328642: Update Autogenerated Commands (#4019)
mongodb/mongodb-atlas-cli: docs/command/atlas-api-teams-getProjectTeam.txt

.. _atlas-api-teams-getProjectTeam:

==============================
atlas api teams getProjectTeam
==============================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

`Public Preview: please provide feedback <https://feedback.mongodb.com/forums/930808-atlas-cli>`_: Returns one team to which the authenticated user has access in the project specified using its unique 24-hexadecimal digit identifier.

> Public Preview: The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).

Reviewer: ```suggestion
Public Preview: The ``atlas api`` sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
```
I think we should consider monospacing this, since it's referring specifically to a command.

Admin API capabilities have their own release lifecycle, which you can check via the provided API endpoint documentation link.

All members of the team share the same project access. To use this resource, the requesting Service Account or API Key must have the Project Read Only role.

This command is autogenerated and corresponds 1:1 with the Atlas API endpoint https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Teams/operation/getProjectTeam.

For more information and examples, see the referenced API documentation linked above.

Syntax
------

.. code-block::
   :caption: Command Syntax

   atlas api teams getProjectTeam [options]

.. Code end marker, please don't delete this comment

Options
-------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - --envelope
     - envelope
     - false

>      - Flag that indicates whether Application wraps the response in an envelope JSON object. Some API clients cannot access the HTTP response headers or status code. To remediate this, set envelope=true in the query. Endpoints that return a list of results use the results object as an envelope. Application adds the status parameter to the response body.

Reviewer: ```suggestion
     - Flag that indicates whether application wraps the response in an envelope JSON object. Some API clients cannot access the HTTP response headers or status code. To remediate this, set ``envelope=true`` in the query. Endpoints that return a list of results use the results object as an envelope. Application adds the status parameter to the response body.
```
Is there a specific reason Application is capitalized in the first instance despite not being the lead word of the sentence?

Reviewer 2: Hi! Fixing `Applicatoin`, but on the second suggestion: we've seen issues with using `` in snooty. Are we sure ``envelope=true``  is the right way or should it be `envelope=true`? (context https://github.com/mongodb/mongodb-atlas-cli/pull/4024)

Reviewer 2: I'll update the envelope too as I realized I was inverting the logic and trying to use ` instead of ``

   * - --groupId
     - string
     - true
     - Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.
       
       **NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.
   * - -h, --help
     - 
     - false
     - help for getProjectTeam
   * - -o, --output
     - string
     - false

>      - preferred api format, can be ["json", go-template] This value defaults to "json".

Reviewer: ```suggestion
     - Preferred API format, can be ["json", go-template] This value defaults to "json".
```
We need to be consistent about capitalization for the opening word. Throughout the CLI docs we have favored capitalizing these, so let's stick to that. As for api -> API, "api" should only be used if we're referencing some specific syntax that uses it in lowercase, otherwise it's an acronym and should be capitalized appropriately.

   * - --outputFile
     - string
     - false

>      - file to write the api output to. This flag is required when the output of an endpoint is binary (ex: gzip) and the command is not piped (ex: atlas command > out.zip)

Reviewer: ```suggestion
     - File to write the API output to. This flag is required when the output of an endpoint is binary (e.g. gzip) and you do not pipe the command (e.g. atlas command > out.zip).
```
Capitalization, punctuation. Favor active voice.

   * - --pretty
     - 
     - false
     - Flag that indicates whether the response body should be in the prettyprint format.
   * - --teamId
     - string
     - true
     - Unique 24-hexadecimal digit string that identifies the team for which you want to get.
   * - --version
     - string
     - false

>      - api version to use when calling the api call [options: "2023-01-01"], defaults to the latest version or the profiles api_version config value if set This value defaults to "2023-01-01".

Reviewer: ```suggestion
     - Version to use when calling the API, formatted as a "YYYY-MM-DD" date string. If you omit this flag, the API uses the version specified in your CLI profile in the ``api_version`` field. If you don't declare ``api_version``, this value defaults to "2023-01-01".
```
One does not "call an API call". The bracketed options aside here also isn't very clear, so let's be more explicit about what it's meant to signal.
